pkg/controller/template/render.go

```diff
@@ -708,3 +743,11 @@ func isOpenShiftManagedDefaultLB(cfg RenderConfig) bool {
 	}
 	return false
 }
+
+func isVSphereCSIMigrationEnabled(cfg RenderConfig) interface{} {
+	if cfg.StorageConfig.Spec.VSphereStorageDriver == operatorv1.CSIWithMigrationDriver {
+		return "enabled"
+	}
```

Comment:
Do you need to handle the case where StorageConfig is nil, similar to `isCloudProviderExternal`?
https://github.com/openshift/machine-config-operator/pull/3655/files#diff-f02b383ddd2e30a342d61ceabe5d3c30742a0f169680cf09a449c3b53ffd6babR420

Reply:
yes, good spot